### PR TITLE
Config file change to load custom configs from a mounted include dir

### PR DESCRIPTION
**Pull request by Red-HAP**
## Description

This branch adds configuration parameters for the database with values taken from a `.env` file which **must** be sourced before launching the local database.

These are updates to `cutwater/misc/pg_stat_statements`

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# =======================================================================
+
+# This is an environment settings file for ansible-events-ui
+
+# =======================================================================
+# =======================================================================
+
+# PostgreSQL General Settings
+# See: https://www.postgresql.org/docs/current/runtime-config-connection.html
+export PG_MAX_CONNECTIONS=1000
```
Comment (Alex-Izquierdo) [inline .env:11]: .env doesn't support "export" keyword

**Review by Alex-Izquierdo — CHANGES_REQUESTED** [review]
When I proposed to move all those configurations to the .env file, I didn't mean add the .env to the repository. 
.env file is precisely to customize the environment and It should not be added to the repository. In fact, it should be added to the .gitignore. 

You could add something like `.env.sample`
Also I think the docs should speak about it. 

What I propose is defining all those configurations in the docker-compose through environment variables with default values. E.g:

```yaml
    volumes:
      - "./postgres/initdb.d:/docker-entrypoint-initdb.d"
      - "postgres_data:/var/lib/postgresql/data"
    environment:
      - PG_MAX_CONNECTIONS=${PG_MAX_CONNECTIONS:-1000}
    command: ->
      postgres -c max_connections=${PG_MAX_CONNECTIONS}
 ```
 
 So, If I want to customize it, I can create a .env file without modify any versioned file. like:
 ```
 PG_MAX_CONNECTIONS=2000
 ```

Also, to keep the compose definition more readable and compact, I suggest use the following syntax:
```yaml
    command: ->
      postgres 
      -c ssl=${PG_SSL}
      -c ssl_cert_file=${PG_SSL_CERT_FILE}
      -c ssl_key_file=${PG_SSL_KEY_FILE}
# etc...
```

**Review by cutwater — CHANGES_REQUESTED** [review]
@Red-HAP Thank you for all your effort working on this change. I really like how you documented the database options and fine tuning that may be required. I have a few questions left, can you please take a look?

File: tools/docker/postgres/initdb.d/uuid-ossp.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+CREATE EXTENSION "uuid-ossp";
```
[the remark this answers is not in the document]
Comment (Red-HAP) [reply tools/docker/postgres/initdb.d/uuid-ossp.sql:1]: I can take it out, but, from my memory, I thought you and I were talking about using uuid_generate_v4() for primary keys.

If I was mistaken, or if this is too much scope-creep, then I'll take it out.

[the remark this answers is not in the document]
Comment (cutwater) [reply tools/docker/postgres/initdb.d/uuid-ossp.sql:1]: At the moment the only place where UUIDs are generated as primary keys is `user` table, and it generates them on a python side.

[the remark this answers is not in the document]
Comment (cutwater) [reply tools/docker/postgres/initdb.d/uuid-ossp.sql:1]: When there is a need for `uuid-ossp` occurs, we can add it in migration.

File: README-POSTGRES.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+#### Logging
+
+* [Logging Documentation](https://www.postgresql.org/docs/current/runtime-config-logging.html)
+
+The following changes were made to the the logging parameters:
+
+| Parameter | Changed Value | Original Value | Description |
+| --------- | ------------- | -------------- | ----------- |
+| logging_collector | **on** | off | Activate (on) or deactivate (off) the logging collector. The logging collector captures stderr logging and writes the data to rotated log files in the `${PGDATA}/log` directory. |
```
[the remark this answers is not in the document]
Comment (Red-HAP) [reply README-POSTGRES.md:44]: This is probably a holdover from working with platforms like AWS. I can take it out... but the question I have is: How much does docker/podman retain from stderr logging? The advantage I can see is that the rotated log files could possible retain more data. If the docker log retention is acceptible, I can take this out.

Thoughts?

[the remark this answers is not in the document]
Comment (cutwater) [reply README-POSTGRES.md:44]: AFAIK docker keeps logs indefinitely for a container _by default_. So as long as a container exists, logs are stored too.
Logging driver and options can be customized though, but personally I've never had a need to do that in my local development environment.

File: tools/docker/postgres/initdb.d/uuid-ossp.sql (hunk 2 above)
[the remark this answers is not in the document]
Comment (Red-HAP) [reply tools/docker/postgres/initdb.d/uuid-ossp.sql:1]: Removed.

File: tools/docker/postgres/postgresql.conf [hunk 4]
```diff
@@ -0,0 +1,792 @@
...
+#------------------------------------------------------------------------------
+# CONFIG FILE INCLUDES
+#------------------------------------------------------------------------------
+
+# These options allow settings to be loaded from files other than the
+# default postgresql.conf.  Note that these are directives, not variable
+# assignments, so they can usefully be given more than once.
+
+#include_dir = '...'			# <== Original value
+include_dir = '/etc/postgresql/conf.d'			# include files ending in '.conf' from
```
[the remark this answers is not in the document]
Comment (Red-HAP) [reply tools/docker/postgres/postgresql.conf:773]: Verified and done.

File: README-POSTGRES.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Red-HAP) [reply README-POSTGRES.md:62]: logging_collector has been reset to off

File: README-POSTGRES.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+| log_min_duration_statement | **2000** | -1 | -1 disables; 0 logs all statements; >0 logs statements executing longer than the specified value (in milliseconds). This is set to log any statement that runs longer than 2 seconds. |
```
[the remark this answers is not in the document]
Comment (Red-HAP) [reply README-POSTGRES.md:63]: Set to 1000

**Review by cutwater — APPROVED** [review]

**Pull request merged**